src/main/java/Session.java

```diff
@@ -6,6 +6,7 @@
 
     private String type;
     private String room;
+    private String section;
```

Comment:
I mentioned this in discord, but for the ICS, also need course code + (lec or tutorial code), to create event titles.